src/main/java/org/influxdb/impl/BatchProcessor.java

```diff
@@ -13,6 +13,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import org.apache.commons.collections4.keyvalue.MultiKey;
```

Comment:
I dont want to introduce more dependencies, a almost equivalent implementation would be:
https://google.github.io/guava/releases/19.0/api/docs/com/google/common/collect/Multimap.html